docs/git-node.md

```diff
@@ -224,6 +224,8 @@ Options:
   --security  Demarcate the new security release as a security release [boolean]
   --startLTS  Mark the release as the transition from Current to LTS   [boolean]
   --filterLabel Filter PR by label when preparing a security release   [string]
+  --yes       Assume "yes" as answer to all prompts and run
+              non-interactively. [boolean] [default: false]
```

Comment:
I'm probably stating the obvious but it will assume "No" when the default answer is a "No". Not a big deal but the wording is somewhat confusing as is

Reply:
hmm, aI thought it would assume `Yes` for all cases. I might need to change all defaults to `yes`.

Reply:
It might still be worth removing the potentially wrong part
```suggestion
  --yes       Skip all prompts and run
              non-interactively. [boolean] [default: false]
```